src/test/test-copy.c

```diff
@@ -50,6 +51,123 @@ TEST(copy_file) {
         unlink(fn_copy);
 }
 
+TEST(copy_tree_replace_file) {
+        _cleanup_free_ char *buf = NULL;
+        char *src, *dest;
```

Comment:
Missing _cleanup_free_ for these (and they can then just be merged into the first declaration)

Reply:
Also, you use `dest` here but `dst` in the second test, I'd be consistent and use the same for both tests